.github/workflows/test-coverage.yml

```diff
@@ -43,7 +36,7 @@ env:
   PYTHONUNBUFFERED: true
 
 jobs:
-  
+
```

Comment:
I cannot see this on my mobile device: is this adding spaces to an otherwise empty line? It should not. If it removes spaces: perfect :-)